### ISSUE TITLE
Timeout Error

### ISSUE THREAD

**Issue by mohcow** [Garulf/BetterTTV-Twitch-Emotes#1]
This Error occured after searching for 10+ seconds. Might be timeout related or something. A restart of Flow Launcher fixed it. 

```
Flow Launcher version: 1.9.1
OS Version: Microsoft Windows NT 10.0.19044.0
IntPtr Length: 8
x64: True

Python Path: 
Date: 01/31/2022 16:31:52
Exception:
System.IO.InvalidDataException: BetterTTV Twitch Emotes|Exception ignored in: <function Launcher.__del__ at 0x0000021F0B494700>
Traceback (most recent call last):
  File "C:\Users\nokz\AppData\Roaming\FlowLauncher\Plugins\BetterTTV-Twitch-Emotes-e87468fc-9fd2-4a6f-9bc6-1faf9a409db8\lib\flox\launcher.py", line 59, in __del__
    self.__call__()
  File "C:\Users\nokz\AppData\Roaming\FlowLauncher\Plugins\BetterTTV-Twitch-Emotes-e87468fc-9fd2-4a6f-9bc6-1faf9a409db8\lib\flox\__init__.py", line 96, in __call__
    return super().__call__()
  File "C:\Users\nokz\AppData\Roaming\FlowLauncher\Plugins\BetterTTV-Twitch-Emotes-e87468fc-9fd2-4a6f-9bc6-1faf9a409db8\lib\flox\launcher.py", line 41, in __call__
    results = request_method(*request_parameters)
  File "C:\Users\nokz\AppData\Roaming\FlowLauncher\Plugins\BetterTTV-Twitch-Emotes-e87468fc-9fd2-4a6f-9bc6-1faf9a409db8\lib\flox\__init__.py", line 127, in _query
    self.query(query)
  File "C:\Users\nokz\AppData\Roaming\FlowLauncher\Plugins\BetterTTV-Twitch-Emotes-e87468fc-9fd2-4a6f-9bc6-1faf9a409db8\plugin\main.py", line 12, in query
    self.result(emote)
  File "C:\Users\nokz\AppData\Roaming\FlowLauncher\Plugins\BetterTTV-Twitch-Emotes-e87468fc-9fd2-4a6f-9bc6-1faf9a409db8\plugin\main.py", line 47, in result
    icon=utils.get_icon(bttv.get_img_url(item), self.name, file_name=f"{item['id']}.{file_ext}"),
  File "C:\Users\nokz\AppData\Roaming\FlowLauncher\Plugins\BetterTTV-Twitch-Emotes-e87468fc-9fd2-4a6f-9bc6-1faf9a409db8\lib\flox\utils.py", line 104, in get_icon
    download_file(url, full_path)
  File "C:\Users\nokz\AppData\Roaming\FlowLauncher\Plugins\BetterTTV-Twitch-Emotes-e87468fc-9fd2-4a6f-9bc6-1faf9a409db8\lib\flox\utils.py", line 85, in download_file
    request.urlretrieve(url, path)
  File "urllib\request.py", line 247, in urlretrieve
  File "urllib\request.py", line 222, in urlopen
  File "urllib\request.py", line 525, in open
  File "urllib\request.py", line 542, in _open
  File "urllib\request.py", line 502, in _call_chain
  File "urllib\request.py", line 1397, in https_open
  File "urllib\request.py", line 1357, in do_open
urllib.error.URLError: <urlopen error _ssl.c:1108: The handshake operation timed out>

   at Flow.Launcher.Core.Plugin.JsonRPCPlugin.ExecuteAsync(ProcessStartInfo startInfo, CancellationToken token) in C:\projects\flow-launcher\Flow.Launcher.Core\Plugin\JsonRPCPlugin.cs:line 308
   at Flow.Launcher.Core.Plugin.JsonRPCPlugin.ExecuteAsync(ProcessStartInfo startInfo, CancellationToken token) in C:\projects\flow-launcher\Flow.Launcher.Core\Plugin\JsonRPCPlugin.cs:line 308
   at Flow.Launcher.Core.Plugin.JsonRPCPlugin.QueryAsync(Query query, CancellationToken token) in C:\projects\flow-launcher\Flow.Launcher.Core\Plugin\JsonRPCPlugin.cs:line 323
   at Flow.Launcher.Core.Plugin.PluginManager.<>c__DisplayClass22_0.<<QueryForPluginAsync>b__0>d.MoveNext() in C:\projects\flow-launcher\Flow.Launcher.Core\Plugin\PluginManager.cs:line 192
--- End of stack trace from previous location ---
   at Flow.Launcher.Infrastructure.Stopwatch.DebugAsync(String message, Func`1 action) in C:\projects\flow-launcher\Flow.Launcher.Infrastructure\Stopwatch.cs:line 40
   at Flow.Launcher.Core.Plugin.PluginManager.QueryForPluginAsync(PluginPair pair, Query query, CancellationToken token) in C:\projects\flow-launcher\Flow.Launcher.Core\Plugin\PluginManager.cs:line 211
   at Flow.Launcher.ViewModel.MainViewModel.<>c__DisplayClass137_0.<<QueryResults>g__QueryTask|2>d.MoveNext() in C:\projects\flow-launcher\Flow.Launcher\ViewModel\MainViewModel.cs:line 623
--- End of stack trace from previous location ---
   at Flow.Launcher.ViewModel.MainViewModel.QueryResults() in C:\projects\flow-launcher\Flow.Launcher\ViewModel\MainViewModel.cs:line 624
   at System.Threading.Tasks.Task.<>c.<ThrowAsync>b__140_0(Object state)
   at System.Windows.Threading.ExceptionWrapper.InternalRealCall(Delegate callback, Object args, Int32 numArgs)
   at System.Windows.Threading.ExceptionWrapper.TryCatchWhen(Object source, Delegate callback, Object args, Int32 numArgs, Delegate catchHandler)
```

**Comment by Garulf** [comment]
Looks like it had the connection timeout when trying to download an emote image. Nothing to worry about. I'll add a catch to skip this error and continue with the rest if possible.

**Comment by Garulf** [comment]
Should be fixed in v0.0.2
